lib/index.js

```diff
@@ -11,5 +12,6 @@ var main = function (options, callback) {
 
 module.exports = main;
 module.exports.launch = main;
+module.exports.argv = argv;
```

Comment:
> Seems like is now really coupling it

In what way? The idea is that `launch` should still expect an options hash, we're just explicitly parsing argv now. ember-cli-sauce is coupled to saucie anyway, isn't that the purpose of it?